### DISCUSSION TITLE
Too high level of detail

### DISCUSSION THREAD

**Discussion by hamogu** [astropy/astropy-v5.0-paper#85] [General comments] [answered]
This is a very long paper and in several places, I have the feeling that the level of detail is too high. I was the referee (but I certainly won't be), I would say "shorten by at least 25%" because I think that can be done relatively easily without sacrificing important content. A few examples:

- Section 5.1: Does it really matter that the items in the roadmap are color-coded and what color has which meaning? Or that we use a "project-board"? What is a project board anyway? That's not important for the reader of this paper.
- Section 4.5: Does it really matter that the website is "gatsby/react with angolia search-as-a-service"? For the average astronomer, the understandable words in that are "with", "as", and "a".  Or who paid for what both in which year? We are writing a scientific paper, not an annual report for the funding agency here.
- Section 4.4.1: Need to list all sub-packages used?

**Comment by eerovaher** [comment]
Regarding Section 4.4.1, even if it is desirable to list all the sub-packages it should be done only once, but right now they are listed twice.

**Reply by adrn** [reply]
Thanks - I did remove the first list.

**Comment by adrn** [comment, marked as answer]
Feel free to open a pull request with proposed trimming of the text. I slightly disagree given the intent of this paper, so am not planning to do this myself before submission.
